go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -227,6 +228,7 @@ func runOvnKube(ctx *cli.Context) error {
 		}
 		watchFactory = masterWatchFactory
 		var ovnNBClient, ovnSBClient goovn.Client
+		var libovsdbOvnNBClient, libovsdbOvnSBClient libovsdbClient.Client
 
 		if ovnNBClient, err = util.NewOVNNBClient(); err != nil {
```

Comment:
we want to get rid of the go-ovn clients?

Reply:
Yes but gradually right? Otherwise we break things or there would be a lot of changes to assume in this PR.

We might want to work on a feature branch if we want to avoid confusion to others not implicated with this work.